static/js/components/EmailCompositionDialog_test.js

```diff
@@ -18,12 +19,16 @@ describe('EmailCompositionDialog', () => {
     closeEmailDialog: sinon.stub().returns(updateStub),
     updateEmailEdit: sinon.stub().returns(updateStub),
     open: true,
-    email: Object.assign({}, NEW_EMAIL_EDIT),
+    email: Object.assign({}, NEW_EMAIL_EDIT.email),
```

Comment:
I'm not sure what this is supposed to do, it looks like `email` doesn't exist on `NEW_EMAIL_EDIT`